server/middleware/auth.js

```diff
@@ -0,0 +1,37 @@
+const passport = require('passport')
+const passportJWT = require('passport-jwt')
+const ExtractJwt = passportJWT.ExtractJwt
+const Strategy = passportJWT.Strategy
+
+// load up the user model
+const User = require('../src/user/model')
+const config = require('../config/auth') // get auth secret key
+
+const params = {
+  secretOrKey: config.jwtSecret,
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken()
+}
+
+module.exports = function() {
+  passport.use(new Strategy(params, function(jwtPayload, done) {
+    User.findOne({ id: jwtPayload.id })
+      .then((user) => {
+        if (user) {
```

Comment:
This `if - else` can be written more simpler; you can write in one line like this: `done(null, user || false)` this means take user if exists or use `false`, this is exactly what the the block does right now.